src/pages/NewVaccineModulePage.js

```diff
@@ -1 +1,192 @@
+        containerStyle={{ flex: 1 }}
+        Icon={<Circle size={20} backgroundColor={FINALISE_GREEN} />}
+      >
+        {description}
+      </TextWithIcon>
+      <IconButton Icon={<DownloadIcon color={BLACK} />} containerStyle={localStyles.iconButton} />
+      <IconButton
+        Icon={<LightbulbIcon color={BLACK} />}
+        onPress={() => blinkSensor(sensor.macAddress)}
+        containerStyle={localStyles.iconButton}
+      />
+      <IconButton
+        Icon={<CogIcon color={BLACK} />}
+        onPress={() => toEditSensorPage(sensor)}
+        containerStyle={localStyles.iconButton}
+      />
+    </>
+  );
+
+  return (
+    <Paper
+      // macAddress passed to fridgeHeader is quite naff. The settings page for the "fridge" is
+      // actually for a sensor, assuming 1 sensor per fridge.
+      Header={header}
+      contentContainerStyle={localStyles.fridgePaperContentContainer}
+    >
+      {sensors.map(({ id, macAddress, batteryLevel, logs, breaches }) => {
+        const mostRecentLog = logs.sorted('timestamp', true)[0];
+        const lastSyncMessage = mostRecentLog ? formatDate(mostRecentLog.timestamp) : 'No logs yet';
+        const temperature = mostRecentLog ? formatTemperature(mostRecentLog.temperature) : 'N/A';
+        const mostRecentBreach = breaches?.sorted('endTimestamp', true)[0];
+        const lastBreachMessage = mostRecentBreach
+          ? formatDate(mostRecentBreach.endTimestamp)
+          : 'Never!';
+
+        return (
+          <FlexRow key={id}>
+            <TextWithIcon left size="ms" containerStyle={{ flex: 1 }} Icon={<Circle size={20} />}>
+              {macAddress}
+            </TextWithIcon>
+            <TextWithIcon
+              left
+              size="ms"
+              containerStyle={localStyles.fridgeDetail}
+              Icon={<TemperatureIcon color={DARKER_GREY} />}
+            >
+              {temperature}
+            </TextWithIcon>
+            <TextWithIcon
+              left
+              containerStyle={localStyles.fridgeDetail}
+              size="ms"
+              Icon={<AlarmClockIcon color={DARKER_GREY} />}
+            >
+              {lastBreachMessage}
+            </TextWithIcon>
+            <TextWithIcon
+              left
+              size="ms"
+              containerStyle={localStyles.fridgeDetail}
+              Icon={<WifiIcon color={DARKER_GREY} />}
+            >
+              {lastSyncMessage}
+            </TextWithIcon>
+            <TextWithIcon
+              left
+              size="ms"
+              containerStyle={localStyles.fridgeDetail}
+              Icon={<BatteryIcon color={DARKER_GREY} />}
+            >
+              {`${batteryLevel}%`}
+            </TextWithIcon>
+            <IconButton
+              Icon={<ChevronRightIcon color={BLACK} />}
+              onPress={() => toFridgeDetail(fridge)}
+              containerStyle={localStyles.iconButton}
+            />
+          </FlexRow>
```

Comment:
Unfortunately the designs, every item had the same amount of characters so wasn't 100% clear if we should make them align. 

I was wondering if it's better to use styles width prop, or a right pad function on the text? I'm thinking a right pad function would be good as it may be more forgiving for different locales?